src/runtime/database.h

```diff
@@ -60,6 +60,15 @@ struct Time {
   std::string as_string() const;
 };
 
+struct RunReflection {
+  int id;
```

Comment:
This is the same data type as in the DB right?

Reply:
Yep!
`CREATE TABLE runs(  run_id  integer primary key autoincrement,  time    integer not null,  cmdline text    not null);`